### PR TITLE
Feature/notification

### PR DESCRIPTION
**Pull request by phoelapyae23**
Please check codes.

### CODE REVIEW

File: logout.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 logout
-hola, amigo by Amy.
\ No newline at end of file
+hola, amigo by Amy.
+Gracias
```
Comment (phyoko72) [inline logout.txt:3]: add some more text.

**Review by phyoko72 — CHANGES_REQUESTED** [review]
Please follow my remarks and push again.

File: logout.txt (hunk 1 above)
Comment (phoelapyae23) [reply logout.txt:3]: ok then.

**Review by phyoko72 — APPROVED** [review]
Ok. Go ahead.

**Pull request merged**